packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -896,6 +900,7 @@ export function getChildFormatContext(
 }
 
 function getSuspenseViewTransition(
+  resumableState: ResumableState,
```

Comment:
why did you add this?

Reply:
I originally thought I'd do it in both branches but then realized that I could do it only if we ever render the fallback.